src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -245,47 +237,45 @@ public void BeginGetRequestStream_CreatePostRequestThenAbort_ThrowsWebException(
         }
 
         [Theory, MemberData(nameof(EchoServers))]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #17842")] //Test hang forever in desktop.
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "no exception thrown on netfx")]
         public void BeginGetRequestStream_CreatePostRequestThenCallTwice_ThrowsInvalidOperationException(Uri remoteServer)
         {
-            _savedHttpWebRequest = HttpWebRequest.CreateHttp(remoteServer);
-            _savedHttpWebRequest.Method = "POST";
+            HttpWebRequest request = HttpWebRequest.CreateHttp(remoteServer);
+            request.Method = "POST";
 
-            IAsyncResult asyncResult = _savedHttpWebRequest.BeginGetRequestStream(null, null);
+            IAsyncResult asyncResult = request.BeginGetRequestStream(null, null);
             Assert.Throws<InvalidOperationException>(() =>
             {
-                _savedHttpWebRequest.BeginGetRequestStream(null, null);
+                request.BeginGetRequestStream(null, null);
             });
         }
 
         [Theory, MemberData(nameof(EchoServers))]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #17842")] //Test hang forever in desktop.
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "request stream not allowed for GET on netfx")]
         public void BeginGetRequestStream_CreateRequestThenBeginGetResponsePrior_ThrowsInvalidOperationException(Uri remoteServer)
         {
-            _savedHttpWebRequest = HttpWebRequest.CreateHttp(remoteServer);
+            HttpWebRequest request = HttpWebRequest.CreateHttp(remoteServer);
 
-            IAsyncResult asyncResult = _savedHttpWebRequest.BeginGetResponse(null, null);
+            IAsyncResult asyncResult = request.BeginGetResponse(null, null);
             Assert.Throws<InvalidOperationException>(() =>
             {
-                _savedHttpWebRequest.BeginGetRequestStream(null, null);
+                request.BeginGetRequestStream(null, null);
             });
```

Comment:
I understand the concern that several of these tests starting network I/O with BeginGetResponse() but don't seem to finish it with EndGetResponse (i.e. via a callback etc.).  I'd rather not complicate this PR by trying to add all of that now.  The tests are currently stable.